packages/inter-protocol/src/vaultFactory/liquidateIncrementally.js

```diff
@@ -197,9 +197,22 @@ const start = async zcf => {
         trace('exiting processTranches loop');
         return;
       }
+      // This await almost certainly does not do what was intended.
```

Comment:
Dang it! That seems correct. How would it get integrated so that a long-lived generator is pulled on and awaited once per block

Reply:
See https://github.com/Agoric/agoric-sdk/issues/6220